packages/tracing-operations/src/utilities/config.ts

```diff
@@ -0,0 +1,25 @@
+import { HTTPServerConfig, LoggerConfig } from "pagopa-interop-tracing-commons";
+import { DbConfig } from "./dbConfig.js";
+import { z } from "zod";
+
+const eServiceOperationsConfig = HTTPServerConfig.and(LoggerConfig)
+  .and(DbConfig)
+  .and(
+    z
+      .object({
+        APPLICATION_NAME: z.string(),
+        SCHEMA_NAME: z.string(),
+        TRACING_DB_USE_SSL: z.coerce.boolean(),
```

Comment:
shouldn't `SCHEMA_NAME` and `TRACING_DB_USE_SSL` go under `DbConfig`?
in case I suggest to rename `TRACING_DB_USE_SSL` in `DB_USE_SSL`